psalm.xml

```diff
@@ -3,7 +3,6 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
-    errorBaseline="psalm-baseline.xml"
```

Comment:
Should we not keep this but leave it empty?

Reply:
Don't see the point in keeping an empty file, we can regenerate it when needed.

Reply:
It is a bit annoying that doesnt look for a default file, like other tools do.
Verbose should only needed for custom names or paths